bittensor/commands/overview.py

```diff
@@ -258,9 +255,9 @@ def _run(cli: "bittensor.cli", subtensor: "bittensor.subtensor"):
                 for neuron in neuron_list:
                     if neuron.hotkey in checked_hotkeys:
                         continue
-                    total_coldkey_stake_from_metagraph[neuron.coldkey] += (
-                        neuron.stake_dict[neuron.coldkey]
-                    )
+                    total_coldkey_stake_from_metagraph[
+                        neuron.coldkey
+                    ] += neuron.stake_dict[neuron.coldkey]
                     checked_hotkeys.add(neuron.hotkey)
 
             alerts_table = Table(show_header=True, header_style="bold magenta")
```

Comment:
subtensor.get_total_stake_for_coldkey below should be awaited. total_coldkey_stake_from_chain is coroutine